### PR TITLE
Lionelb/refactor api add ci

### PR DESCRIPTION
**Pull request by lionelB**
- first step to have API e2e tests
I've introduced 2 changes on the api so to configure elasticsearch indices. 
the script `code-du-travail-api/tests/create_indexes.js` will populate an indexes dedicated to test (indices name finished with `_test`,
So now when launching tests, a local elastic search is needed in order to snapshot API response.

@douglasduteil let the magic happen, your turn to make it green 😋

### CODE REVIEW

File: packages/code-du-travail-data/dataset/synonyms/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (douglasduteil) [inline packages/code-du-travail-data/dataset/synonyms/package.json]: Can be another PR

File: packages/code-du-travail-data/dataset/synonyms/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const synonyms = require("../synonyms.json");
```
Comment (douglasduteil) [inline packages/code-du-travail-data/dataset/synonyms/index.js]: Can be another PR

**Review by douglasduteil — CHANGES_REQUESTED** [review]

File: packages/code-du-travail-api/tests/create_indexes.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+const client = require("../src/server/conf/elasticsearch");
+const { analyzer, filter, tokenizer } = require("./es_analysis");
+const { logger } = require("../src/server/utils/logger");
+const documentsData = require("./cdtn_document_data.json");
```
Comment (douglasduteil) [inline packages/code-du-travail-api/tests/create_indexes.js:5]: ? is it `cdtn_document_data.js`

**Review by douglasduteil — CHANGES_REQUESTED** [review]

File: packages/code-du-travail-api/tests/es_analysis.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
+const synonyms = require("@cdt/data...synonyms");
```
Comment (douglasduteil) [inline packages/code-du-travail-api/tests/es_analysis.js:1]: Should be removed now ?

**Review by douglasduteil — CHANGES_REQUESTED** [review]

File: packages/code-du-travail-api/tests/es_analysis.js (hunk 4 above)
Comment (lionelB) [reply packages/code-du-travail-api/tests/es_analysis.js:1]: we can merge the next PR before 🙈

**Comment by douglasduteil** [comment]
:champagne:

**Review by douglasduteil — APPROVED** [review]
